pkg/registry/endpoint/etcd/etcd.go

```diff
@@ -33,7 +33,7 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against endpoints.
-func NewStorage(h tools.EtcdHelper) *REST {
+func NewStorage(s tools.StorageInterface) *REST {
```

Comment:
I am SO HAPPY to see this change.  Now rename etcdgeneric and rename all of the pkg/registry/*/etcd to something else :)  I don't want to see "etcd" anywhere in the code base outside of one or two packages that implement our storage API in terms of etcd.